schemas/list-workertypes-response.yml

```diff
@@ -20,10 +20,45 @@ properties:
           minLength:    {$const: identifier-min-length}
           maxLength:    {$const: identifier-max-length}
           pattern:      {$const: identifier-pattern}
+        provisionerId:
+          title:        "Provisioner ID"
+          type:         string
+          minLength:    {$const: identifier-min-length}
+          maxLength:    {$const: identifier-max-length}
+          pattern:      {$const: identifier-pattern}
+        stability:
+          title:        "Stability"
+          description: |
+            The stability of the worker-type. Accepted values:
+            'experimental|stable|deprecated'
+          type:         string
+          enum:         ["experimental", "stable", "deprecated"]
+        description:
+          title:        "Description"
+          description: |
+            Description of the worker-type.
+          type:         string
+        documentation:
```

Comment:
Per IRC discussion, let's drop documentation and schema entirely, rather than continue to worry about them here.  We can come back in a later PR and add them, hopefully only in the get-workertype-response, and possibly in an azure blob.

Description and stability and expires can stay.  `provisionerId`, also added here, seems a little redundant but it doesn't hurt.